ewkcoffea/modules/objects_Run3_2Lep.py

```diff
@@ -8,36 +8,47 @@ def get_cleaned_collection(obj_collection_a,obj_collection_b,drcut=0.4):
     mask = ak.fill_none(dr>drcut,True)
     return mask
 
-# Run3 2Lep preselection for electrons
-def is_presel_Run3_2Lep_ele(ele):
+def is_veto_Run3_2Lep_ele(ele):
```

Comment:
Same comment as before on function names (would be good for all new functions we add to try to follow python naming conventions of lower case letters).